docs/spec/auth/07_params.md

```diff
@@ -0,0 +1,11 @@
+# Parameters
+
+The auth module contains the following parameters:
+
+| Key                    | Type            | Example |
```

Comment:
rather than `Example` why not include `Default`?

Reply:
These aren't the defaults necessarily (I got these for a local testnet). I think examples are better as defaults may be subject to change and would be better for readers to understand.

Reply:
sure